plugins/samanage/plugin.spec.yaml

```diff
@@ -123,60 +125,60 @@ types:
     category:
       title: Category
       description: Category
-      type: samanage_name
+      type: solarwinds_name
     subcategory:
       title: Subcategory
       description: Subcategory
-      type: samanage_name
+      type: solarwinds_name
     requester:
       title: Requester
       description: Requester
-      type: samanage_email
+      type: solarwinds_email
     due_at:
       title: Due At
       description: Due at
       type: string
     assignee:
       title: Assignee
       description: Assignee
-      type: samanage_email
+      type: solarwinds_email
     problem:
       title: Problem
       description: Problem
-      type: samanage_problem
+      type: '[]solarwinds_problem'
     changes:
       title: Changes
       description: Changes
-      type: '[]samanage_number'
+      type: '[]solarwinds_number'
     assets:
       title: Assets
       description: Assets
-      type: '[]samanage_id'
+      type: '[]solarwinds_id'
     mobiles:
       title: Mobiles
       description: Mobiles
-      type: '[]samanage_id'
+      type: '[]solarwinds_id'
     configuration_items:
       title: Configuration Items
       description: Configuration items
-      type: '[]samanage_id'
+      type: '[]solarwinds_id'
     incidents:
       title: Incidents
       description: Incidents
-      type: '[]samanage_number'
+      type: '[]solarwinds_number'
```

Comment:
It may be broken typing. 
For `assign_incident` action  output looks different. 
For what we have in typing it looks like: 
```
{"assignee": {"email": "string"}}
```
example output for that action looks like: 
```
"assignee": {
      "group_id": 4485265,
      "is_user": true,
      "id": 4238379,
      "name": "Example User",
      "disabled": false,
      "email": "user@example.com",
      "created_at": "2018-11-20T05:29:00.000-05:00",
      "last_login": "2018-11-20T05:33:33.000-05:00",
      "phone": "(800) 555-0100"
      ...
```

It may be a validation schema error during the tests, but check it out please

Reply:
@llaszuk-r7 , the assignee passed in as an input is just the email address, however the assignee as an output has a number of fields. I noticed that the assignee output has more fields so I've updated it to contain additional fields such as the avatar. Does this address your concern?

Reply:
yes, thank you